The nightly build job failed on Thursday (2023-12-21)

The nightly build job failed on Thursday (2023-12-21) in run [7295697061](https://github.com/TileDB-Inc/TileDB-VCF/actions/runs/7295697061)

The nightly osx-libtiledb-release-2.18 build also failed on my fork. Do we know what this error is about? It's surprising it happened on the release and not dev build

Also, let's try to close these Issues once they are resolved so that new Issues will be opened for new problems

```
==================================== ERRORS ====================================
___________________ ERROR collecting tests/test_tiledbvcf.py ___________________
ImportError while importing test module '/Users/runner/work/TileDB-VCF/TileDB-VCF/TileDB-VCF/apis/python/tests/test_tiledbvcf.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/Users/runner/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_tiledbvcf.py:11: in <module>
    import tiledb
/Users/runner/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/tiledb/__init__.py:39: in <module>
    from .array_schema import ArraySchema
/Users/runner/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/tiledb/array_schema.py:8: in <module>
    import tiledb.cc as lt
E   ImportError: dlopen(/Users/runner/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/tiledb/cc.cpython-310-darwin.so, 0x0002): Symbol not found: (_tiledb_group_get_member_by_index_v2)
E     Referenced from: '/Users/runner/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/tiledb/cc.cpython-310-darwin.so'
E     Expected in: '/Users/runner/work/TileDB-VCF/TileDB-VCF/install/lib/libtiledb.dylib'
```

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        libtiledb_version: [release-2.18, dev]
+        libtiledb_version: [release-2.19, dev]
     steps:
       - name: Clone TileDB
         uses: actions/checkout@v3
@@ -65,7 +65,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        libtiledb_version: [release-2.18, dev]
+        libtiledb_version: [release-2.19, dev]
     env:
       MACOSX_DEPLOYMENT_TARGET: 10.15
     steps:

